extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -61,13 +65,28 @@ export class AmpTimeAgo extends AMP.BaseElement {
 
     this.setFuzzyTimestampValue_();
     this.element.appendChild(this.timeElement_);
+
+    this.viewportObserver_ = getViewportObserver((records) => {
```

Comment:
note: would we prefer one inob per timeago component, or one to track all timeago components per win?

Reply:
This is a good general question across all components. I'd lean toward the default pattern of one inob per component type.

Reply:
I think one per-type might be easier for now. But I think in the long-term the best strategy is one per use-case, i.e. where conflicts between observers cannot happen. That being said, the timeago definitely seems like a separate use case to me. So in this particular case, per-type == per-use-case.